octree/include/pcl/octree/impl/octree2buf_base.hpp

```diff
@@ -39,6 +39,8 @@
 #ifndef PCL_OCTREE_2BUF_BASE_HPP
 #define PCL_OCTREE_2BUF_BASE_HPP
 
+#include <pcl/octree/octree2buf_base.h>
```

Comment:
Yeah, you might be right. Still compiles without including anything here.
I'll remove it.